public/friendlycode/js/fc/ui/bramble-proxy.js

```diff
@@ -19,7 +19,35 @@ define(["backbone-events", "fc/prefs"], function(BackboneEvents, Preferences) {
     var latestSource = "(none)";
     var prefSize = Preferences.get("textSize");
     var that = this;
-    
+    var lastLine = 0;
+    var scrollInfo;
+
+    function communicateEditMessage(fn) {
+      var argLen = arguments.length;
+      var callback = typeof arguments[argLen - 1] === "function" ? arguments[argLen - 1] : undefined;
+      var params = Array.prototype.slice.call(arguments, 1, callback ? argLen - 1 : argLen);
+
+      if(callback) {
+        window.addEventListener("message", function editReceiver(e) {
+          var message = JSON.parse(e.data);
+
+          if(message.type !== "bramble:edit" || message.fn !== fn) {
+            return;
+          }
+
+          window.removeEventListener("message", editReceiver);
```

Comment:
This works? I thought that a function passed as a parameter was always anonymous.